jarbas/dashboard/admin/list_filters.py

```diff
@@ -63,6 +63,23 @@ def queryset(self, request, queryset):
         return filter_option.get(self.value(), queryset)
 
 
+class HasReimbursementNumberFilter(JarbasListFilter):
+
+    title = 'ter número do reembolso'
```

Comment:
I think that we need a better title here, maybe something like `Pago direto ao fornecedor`.

Reply:
The leading _Por_ is fixed in Django, so I changed to _Por pagamento feito direto ao fornecedor_ <3